### PR TITLE
Replace print by echo

### PR DESCRIPTION
**Pull request by realerikrani**
https://click.palletsprojects.com/en/7.x/quickstart/#echoing seems like a preferred way for output.

### CODE REVIEW

File: cli/create.py [hunk 1]
```diff
@@ -29,12 +29,12 @@ def handle_rsa_keys(total: int) -> Tuple[RsaKey, RsaKey, List[str]]:
...
     and retrieves all the other horcruxes RSA public keys.
     """
-    print("Generating RSA key for communicating with other horcruxes...")
+    click.echo("Generating RSA key for communicating with other horcruxes...")
     my_rsa_private_key = RSA.generate(4096)
     my_rsa_public_key = my_rsa_private_key.publickey()
 
-    print(f'\n\n{my_rsa_public_key.export_key("OpenSSH").decode("ascii")}')
-    print("\n\nShare the RSA public key above with all other horcruxes")
+    click.echo(f'\n\n{my_rsa_public_key.export_key("OpenSSH").decode("ascii")}')
```
Comment (tsudmi) [inline cli/create.py]: I think we can use a different color here to emphasize the public key:
```suggestion
    click.secho(f'\n\n{my_rsa_public_key.export_key("OpenSSH").decode("ascii")}', fg="green")
```

File: cli/create.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def handle_dispatcher(
             os.mkdir(DATA_DIR)
         with open(dispatcher_input_file, "w") as dispatcher_file:
             json.dump(input_data, dispatcher_file)
-        print(
+        click.echo(
             f"Saved dispatcher input to {dispatcher_input_file}. "
```
Comment (tsudmi) [inline cli/create.py]:
```suggestion
            "Saved dispatcher input to " + click.style(dispatcher_input_file, fg="green")
```

File: cli/create.py [hunk 3]
```diff
@@ -176,11 +176,11 @@ def process_dispatcher_output(
         horcrux_private_key_shares.append(horcrux_private_key_share)
 
     final_public_key = bls_pop._AggregatePKs(final_public_key_shares)
-    print(f"Shared BLS Public Key: 0x{final_public_key.hex()}")
+    click.echo(f"Shared BLS Public Key: 0x{final_public_key.hex()}")
```
Comment (tsudmi) [inline cli/create.py]: Here and in other places let's choose a color to emphasize the output keys, file paths, etc.
I think the green could fit well.

File: cli/handle_dispatcher.py [hunk 4]
```diff
@@ -19,13 +19,13 @@ def submit_dispatcher_data(
     for data in dispatcher_data:
         response = requests.post(dispatcher_endpoint, json=data)
         if response.status_code != 200:
-            print(
+            click.echo(
                 "Failed to submit dispatcher data: "
```
Comment (tsudmi) [inline cli/handle_dispatcher.py:23]: This one is an error, let's change it's color to red.

File: cli/verify_signature.py [hunk 5]
```diff
@@ -47,6 +47,6 @@ def verify_signature(public_key: str, signing_data: str, signature: str) -> None
     if bls_pop.Verify(
         PK=bls_public_key, message=bytes.fromhex(signing_data), signature=bls_signature
     ):
-        print("[+] The signature is valid")
+        click.echo("[+] The signature is valid")
```
Comment (tsudmi) [inline cli/verify_signature.py]: Lets change the color here to green
```suggestion
        click.secho("[+] The signature is valid", fg="green")
```

File: cli/verify_signature.py [hunk 6]
```diff
@@ -47,6 +47,6 @@ def verify_signature(public_key: str, signing_data: str, signature: str) -> None
...
     else:
-        print("[-] The signature is invalid")
+        click.echo("[-] The signature is invalid")
```
Comment (tsudmi) [inline cli/verify_signature.py]:
```suggestion
        click.secho("[-] The signature is invalid", fg="red")
```

**Review by tsudmi — COMMENTED** [review]
To support colors on Windows we must also install the `colograma` package according to the docs: https://click.palletsprojects.com/en/7.x/utils/#ansi-colors

File: cli/handle_dispatcher.py (hunk 4 above)
Comment (realerikrani) [reply cli/handle_dispatcher.py:23]: a1f1095

**Comment by realerikrani** [comment]
Added colorama in 1a9fe1d.

File: cli/create.py (hunk 3 above)
Comment (realerikrani) [reply cli/create.py]: 684b19b

**Comment by realerikrani** [comment]
Addressed comments, re-requested @tsudmi 's review.

File: cli/handle_dispatcher.py [hunk 7]
```diff
@@ -62,8 +63,11 @@ def poll_dispatcher(
 
         with open(dispatcher_output_file, "w") as dispatcher_file:
             json.dump(output_data, dispatcher_file)
-        print(f"Saved dispatcher output data to {dispatcher_output_file}")
-        print("Move it to your offline machine to process.")
+        click.echo(
+            "Saved dispatcher output data to"
+            f"{click.style(f'{dispatcher_output_file}', fg='green')}"
```
Comment (tsudmi) [inline cli/handle_dispatcher.py:68]:
```suggestion
            f" {click.style(f'{dispatcher_output_file}', fg='green')}"
```

File: cli/handle_dispatcher.py [hunk 8]
```diff
@@ -62,8 +63,11 @@ def poll_dispatcher(
 
         with open(dispatcher_output_file, "w") as dispatcher_file:
             json.dump(output_data, dispatcher_file)
-        print(f"Saved dispatcher output data to {dispatcher_output_file}")
-        print("Move it to your offline machine to process.")
+        click.echo(
+            "Saved dispatcher output data to"
```
Comment (tsudmi) [inline cli/handle_dispatcher.py]:
```suggestion
            "Saved dispatcher output data to "
```

**Review by tsudmi — APPROVED** [review]

**Pull request merged**